README.md

```diff
@@ -18,7 +26,7 @@ sudo apt install libswscale-dev libavcodec-dev libavutil-dev python3-virtualenv
 ## Cloning the repository
```

Comment:
Get rid of additional packages to install, as they are not needed now (they were prerequisites) used by H264Decoder.

Reply:
I'm not sure if it will work due to djitellopy package use tH264Decoder too. I'll delete and we will see :dancers: